src/report_reader.cpp

```diff
@@ -433,11 +433,11 @@ DataFrame<T> ReportReader<T>::Population::get(const nonstd::optional<Selection>&
     // Fill times
     DataFrame<T> data_frame;
     for (size_t i = index_start; i <= index_stop; i += stride) {
-        data_frame.times.push_back(times_index_[i].second);
+        data_frame.times.emplace_back(times_index_[i].second);
     }
 
     // Fill ids
-    data_frame.ids = node_id_element_layout.ids;
+    data_frame.ids.swap(node_id_element_layout.ids);
```

Comment:
it's funny, I had this, but the asm turned out the same in both cases; now I'm doubting that.